packages/ember-glimmer/lib/syntax/render.ts

```diff
@@ -128,12 +132,17 @@ function makeComponentDefinition(vm: VM, args: Arguments) {
   @public
   @deprecated Use a component instead
 */
-export function renderMacro(_name: string, params: any[], hash: any[], builder: any) {
+export function renderMacro(_name: string, params: Option<WireFormat.Core.Params>, hash: WireFormat.Core.Hash, builder: OpcodeBuilderDSL): boolean {
   if (!params) {
     params = [];
   }
-  let definitionArgs = [params.slice(0), hash, null, null];
-  let args = [params.slice(1), hashToArgs(hash), null, null];
-  builder.component.dynamic(definitionArgs, makeComponentDefinition, args);
+  let definitionArgs: ComponentArgs = [params.slice(0), hash, null, null];
+  let args: ComponentArgs = [params.slice(1), hashToArgs(hash), null, null];
+
+  // We have to coerce makeComponentDefinition into an `any` here because
+  // Glimmer VM is typing this too restrictively as a PathReference when it
```

Comment:
Can you cross link to the issue in glimmer-vm that you will be opening to track removal of the coercion?

Reply:
This API no longer exists in Glimmer VM, so I don't think I will be opening that issue.

Reply:
@tomdale - So a future update of glimmer-vm will remove this comment (and what it is doing)?  Can we link to _anything_ that will help us understand this when we come back to this code in 6 months? I fear without any pointers to additional things to dig into, this will become yet another murder mystery for our future selves to solve. 🤔

Reply:
@rwjblue It is already removed in @smfoote's upgrade branch.

Reply:
Awesome, thank you (sorry for the run around).